serverless/pages/manage-org.asciidoc

```diff
@@ -1,15 +1,149 @@
+* <<general-join-organization-from-existing-cloud-account>>: Join a new organization and bring over your projects.
+* <<general-leave-an-organization>>: Leave an organization.
+
+[discrete]
+[[general-manage-access-to-organization]]
+== Invite your team
+
+// :description: Add members to your organization and projects.
+// :keywords: serverless, general, organization, overview
+
+To allow other users to interact with your projects, you must invite them to join your organization and grant them access to your organization resources and instances.
+
+Alternatively, {cloud}/ec-saml-sso.html[configure {ecloud} SAML SSO] to enable your organization members to join the {ecloud} organization automatically. preview:[]
+
+. Go to the user icon on the header bar and select **Organization**.
+. On the **Members** page, click **Invite members**.
+. Enter the email addresses of the users you want to invite in the textbox.
++
+You can add multiple members by entering their email addresses separated by a space or pressing Enter.
++
+You can grant access to all projects of the same type with a unique role, or select individual roles for specific projects.
+For more details about roles, refer to <<general-assign-user-roles>>.
+. Click **Send invites**.
++
+Invitations to join an organization are sent by email. Invited users have 72 hours to accept the invitation before it expires. If the invite has expired, an admin can resend the invitation.
+
+On the **Members** tab of the **Organization** page, you can view the list of current members, their status and role.
+
+In the **Actions** column, click the three dots to edit a member's role or revoke the invite.
+
+[discrete]
+[[general-assign-user-roles]]
+== Assign user roles and privileges
+
+// :description: Manage the predefined set of roles and privileges for all your projects.
+// :keywords: serverless, general, organization, roles, how to
+
+preview:[]
+
+Within an organization, users can have one or more roles and each role grants specific privileges.
+
+You must assign user roles when you <<general-manage-access-to-organization,invite users to join your organization>>.
+To subsequently edit the roles assigned to a user:
+
+. Go to the user icon on the header bar and select **Organization**.
+. Find the user on the **Members** tab of the **Organization** page. Click the member name to view and edit its roles.
+
+[discrete]
+[[general-assign-user-roles-organization-level-roles]]
+=== Organization-level roles
+
+* **Organization owner**. Can manage all roles under the organization and has full access to all serverless projects, organization-level details, billing details, and subscription levels. This role is assigned by default to the person who created the organization.
+* **Billing admin**. Has access to all invoices and payment methods. Can make subscription changes.
+
+[discrete]
+[[general-assign-user-roles-instance-access-roles]]
+=== Instance access roles
```

Comment:
What does instance access mean?

Reply:
"Instance access" is the term used in the Roles UI to refer to "hosted deployment or serverless project".

Reply:
gave additional upfront guidance to clear up the confusion.